packages/nx/src/hasher/hasher.ts

```diff
@@ -156,7 +160,10 @@ export class Hasher {
    * @deprecated use hashTask
    */
   async hashSource(task: Task): Promise<string> {
-    const hash = await this.taskHasher.hashTask(task, [task.target.project]);
+    const hash = await this.taskHasher.hashTask(
+      task,
+      new Set([task.target.project])
+    );
     return hash.details[`${task.target.project}:$filesets`];
```

Comment:
This is probably broken since [this](https://github.com/nrwl/nx/commit/438e4a544e500d3469ef9aba6ed099e4b0960b52) change.